### PR TITLE
Update double-key-encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that Keyname is Case sensitive as it needs to be as in the appsettings.json.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3785

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -452,6 +452,9 @@ key_store_tester.ps1 https://mydkeservice.com/mykey
 
 Ensure that no errors appear in the output. When you're ready, [register your key store](#register-your-key-store).
 
+> [!NOTE]
+> Key name is case senstive. Provide the Keyname as in the appsettings.json.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
> The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Comment by KCCross** [comment]
Remove the "Note" formatting. This is not information they can skip, it's part of the regular flow of information in the text. Do that and I'll merge this.

File: microsoft-365/compliance/double-key-encryption.md [hunk 2]
```diff
@@ -452,6 +452,9 @@ key_store_tester.ps1 https://mydkeservice.com/mykey
 
 Ensure that no errors appear in the output. When you're ready, [register your key store](#register-your-key-store).
 
+> [!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
```

File: microsoft-365/compliance/double-key-encryption.md [hunk 3]
```diff
@@ -452,6 +452,9 @@ key_store_tester.ps1 https://mydkeservice.com/mykey
...
+> The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/double-key-encryption.md]: As per @KCCross's suggestion.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**